src/main/java/org/symphonyoss/symphony/messageml/elements/PersonSelector.java

```diff
@@ -28,19 +31,22 @@
   private static final String PLACEHOLDER_ATTR = "placeholder";
   private static final String REQUIRED_ATTR = "required";
   private static final Set<String> VALID_VALUES_FOR_REQUIRED_ATTR = new HashSet<>(Arrays.asList("true", "false"));
-  private static final String SELECTED_USERS_IDS = "selected-users-ids";
-  private static final String SELECTED_USERS_IDS_FORMAT = "^\\d+(,\\d+)*$";
+  private static final String VALUE_ATTR = "value";
 
   private static final String PRESENTATIONML_TAG = "div";
   private static final String PRESENTATIONML_PLACEHOLDER_ATTR = "data-placeholder";
   private static final String PRESENTATIONML_REQUIRED_ATTR = "data-required";
+  private static final String PRESENTATIONML_VALUE_ATTR = "data-value";
 
   private final static String MARKDOWN = "Person Selector";
   private static final String CLASS_ATTR = "class";
   private static final String PRESENTATIONML_NAME_ATTR = "data-name";
 
+  private final ObjectMapper mapper;
+
   public PersonSelector(Element parent, FormatEnum messageFormat) {
     super(parent, MESSAGEML_TAG, messageFormat);
+    mapper = new ObjectMapper();
```

Comment:
can it be instanciated multiple times? if so the mapper could be static final (it should be thread safe) so it can be reused (it is a bit costly to create)

Reply:
made it static!